src/ppx/ppx_eliom_client.ml

```diff
@@ -152,9 +163,10 @@ module Pass = struct
       open_client_section loc ::
       register_client_closures client_expr_data @
       define_client_functions loc client_expr_data @
-      [ item ;
-        close_server_section loc ;
-      ]
+      [ item ] @
+      (if flush_injection () then
```

Comment:
The `open_client_section` should be removed. Not the `close_server_section`. Right?